index.html

```diff
@@ -0,0 +1,233 @@
+                      <b class="sec3p">.</b>
+                    <a href="/someurl" class="sec3rlink31">2015</a>
+                  </li>
+               </ul>
+
+     <p class="sec3p1 l1"> A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+              <ul class="tm">
+
+               <li class="sec3rlink23">
+                    <a href="/someurl" class="sec3rlink31 slink">HTML</a>
+                    <a href="/someurl" class="sec3rlink31 slink">CSS</a>
+                    <a href="/someurl" class="sec3rlink31  slink">JAVASCRIPT</a>
+                </li>
+              </ul>
+     <button type="submit" class=" bb sec3rbutton sec3rlink4 tm"><span class="d">See Projects</span></button>
+   </section>
+
+
+   
+  <section class="section7">
+               <h2 class="h2 imelex">About <br>Myself</h2>
+                 <h4  class="sec3p1">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</h4>
+                <h5 class="ets" >ETS CONNECT</h5>
+        <ul class="social-menue">
+          <li><img src="images/twitter.PNG" alt="abc"></li>
+          <li><img src="images/linkedinicon.PNG" alt="abc"></li>
+          <li> <img src="images/medium.PNG" alt="abc"></li>
+          <li> <img src="images/github.PNG" alt="abc"></li>
+          <li><img src="images/angellist.PNG" alt="abc"></li>
+        </ul>
+             
+      <button type="submit" class=" bb sec3rbutton sec3rlink4 tm bd"><span class="d">GET my resume</span></button>
+
+               <ul class="limage lsdf">
+                    <li><img src="images/dm.PNG" alt="abc" class="lsdks"></li>
+                    <li><img src="images/k.PNG" alt="abc" class="lsdks"></li>
+                    <li><img src="images/l.PNG" alt="abc" class="lsdks"></li>
+                    <li><img src="images/p.PNG" alt="abc" class="lsdks"></li>
+                    <li><img src="images/q.PNG" alt="abc" class="lsdks"></li>
+                    <li><img src="images/s.PNG" alt="abc" class="lsdks"></li>
+               </ul>
+    </section>
+    
+
+    <section class="sec1">
+        <div class="and">
+        <img src="images/Ellipse12.png" alt="" class="img1">
+    </div>
+        <h2 class="head">contact me</h2>
+        <p class="paragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding.
+             I’d love to help with it</p>
+
+             <form action="abc.php" method="post">
+                <input type="text" placeholder="Aziz" class="name" minlength="30" maxlength="30" required>
+                <input type="email" placeholder="aziz123@gmail.com" class="em">
+                <textarea name="text" id="ad" cols="30" rows="10" maxlength="500" class="message"></textarea>
+                 <button class="sub">Get It Touch</button>
```

Comment:
- [ ] Please add the correct styling to this submit button as it is in the Figma design. In addition, remember to add styles based on different user interactions like `hover`, `pressed`, etc